### PR TITLE
UPP-3256: Currency rate change support on the pricing page

### PR DESCRIPTION
**Pull request by Souler**

### CODE REVIEW

File: app/scripts/main/currency.js [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+      */
+    var CURRENCY_BASE = 'EUR'
+    /**
+     * Other currencies that are not the base and should be handled by this
+     * script.
+     * @type {CurrencyCode[]}
+     */
+    var CURRENCY_SYMBOLS = [ 'USD', 'MXN' ]
+
+    function IpGeolocationService() {}
```
Comment (jarnaiz) [inline app/scripts/main/currency.js]: delete it please, we are not using it

File: app/scripts/main/currency.js [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+        var priceContainers = initializePriceContainers()
+        if (priceContainers.length <= 0) // If no price containers, do nothing
+            return
+        // Retrieve conversion rates and update price containers
+        currencyService.getRates(function(data) {
+            updatePriceContainers(priceContainers, data)
+        })
+        // Retrieve client info by its current IP
+        geolocationService.getInfo(function(data) {
```
Comment (jarnaiz) [inline app/scripts/main/currency.js]: we use the data-lang attr to check the country user

File: package.json [hunk 3]
```diff
@@ -2,6 +2,10 @@
   "name": "landing-upplication",
   "version": "3.6.6",
   "author": "Upplication Team <javier.arnaiz@upplication.com>",
+  "scripts": {
+    "start": "gulp",
+    "deploy": "gulp deploy"
+  },
   "contributors": [
     {
       "name": "Raul Jiménez",
```
Comment (jarnaiz) [inline package.json]: Add @Souler to the contributors section, if you want :)

File: app/scripts/main/currency.js [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+     * provided on service constructon.
+     *
+     * @param  {Function} cb
+     * @return {CurrencyRates}
+     * @see {@link CurrencyService}
+     */
+    CurrencyService.prototype.getRates = function(cb) {
+        var conf = $.extend({}, this.conf)
+
+        var url = 'http://api.fixer.io/latest'
```
Comment (jarnaiz) [inline app/scripts/main/currency.js]: can be https? or maybe //api.fixer.io/latest

**Comment by Souler** [comment]
I leave here two more suggestions that were made by @jarnaiz via JIRA
* Decimal separators used to be commas(,), but now are dot (.)
* The currency switcher doesn't show the currently selected one

File: app/scripts/main/currency.js (hunk 4 above)
Comment (Souler) [inline app/scripts/main/currency.js]: All fixed on 652b8b1

**Pull request merged**